solidity/README.md

```diff
@@ -6,7 +6,6 @@ On-chain implementations of Hyperlane in Solidity.
 
 - `npm install --dev`
```

Comment:
Should these use `yarn` for consistency with other Readmes?

Reply:
Hmm I don't know why any ts was in the diff.